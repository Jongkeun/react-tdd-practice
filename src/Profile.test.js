import React from "react";
import { render } from "@testing-library/react";
import Profile from "./Profile";

describe("<Profile/>", () => {
  it("matches snapshot", () => {
    const utils = render(<Profile username="jongkeun" name="홍종근" />);
    expect(utils.container).toMatchSnapshot();
  });

  it("shows the props conrrectly", () => {
    const utils = render(<Profile username="jongkeun" name="홍종근" />);
    utils.getByText("jongkeun");
    utils.getByText("(홍종근)");
    utils.getByText(/홍/);
  });
});
