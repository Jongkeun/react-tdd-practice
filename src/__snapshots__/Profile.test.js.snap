// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> matches snapshot 1`] = `
<div>
  <div>
    <b>
      jongkeun
    </b>
     
    <span>
      (
      홍종근
      )
    </span>
  </div>
</div>
`;
